conf.py

```diff
@@ -150,31 +150,21 @@
 # (Bootstrap 4: right-side of navbar, Bootblog 4: right side of title)
 NAVIGATION_ALT_LINKS = {
     DEFAULT_LANG: (
+        ("https://docs.ansible.com/", "Documentation", ""),
```

Comment:
This now a problematic link as this site is no longer a community site. Either change to Community documentation or remove it entirely.

This is a problem in other places as well where we point to docs.ansible.com but not to AAP docs on redhat.com.  Not sure how you want to address those problems.

Reply:
Good point and I will see how they wish to address this concern.  My guess is the label will change.

Reply:
@samccann The concern reported as been resolved with a dropdown menu for documentation.